### DISCUSSION TITLE
[4.0] getting npm error for tarball on running npm ci

### DISCUSSION THREAD

**Discussion by pinak3748** [joomla/joomla-cms#34625] [Q&A]
I am new to open source and i tried many things but nothing is working so final i have to start a discussion.
I am getting many warning on running npm ci, i have ran composer install and it has showing successful. 
![Screenshot 2021-06-24 200341](https://user-images.githubusercontent.com/59458280/123281484-5adff680-d527-11eb-8308-1897b26ed9aa.png)

**Comment by brianteeman** [comment]
As the error says in your screenshot - look in the log file

**Reply by pinak3748** [reply]
Thanks the problem as been resolved there was some problem with npm cache
